invenio_cli/cli.py

```diff
@@ -144,9 +144,11 @@ def update(install_js):
               help='Which environment to build, it defaults to local')
 @click.option('--verbose', default=False, is_flag=True, required=False,
               help='Verbose mode will show all logs in the console.')
-def destroy(local, verbose):
+def destroy(local, verbose, remove):
```

Comment:
There is no `remove` option above + this parameter is never used in the function.

```suggestion
def destroy(local, verbose):
```

Reply:
Forgot to remove! Thank you!